CONTRIBUTING.md

```diff
@@ -14,38 +14,49 @@ When contributing to this repository, please first discuss the change you wish t
 
 - In your fork, create a branch for your feature
 
-   ```git checkout -b add-contributing-section-to-readme```
+   ```git checkout -b add-test-ods-542-alerts```
 
 - Develop your feature and create a git commit:
+  - Before committing the code, verify that it is formatted following the _ODS-CI Robot Framework Style Guide_ (see [how](https://github.com/red-hat-data-services/ods-ci/blob/master/docs/check-code-style.md)
+)
   - Add your changes to the commit
-
-     ```git add README.md```
+    ```
+    git add tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot
+    ```
 
    - Sign off your commit using the -s, --signoff option. Write a good commit message (see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/))
-
-
-     ```git commit -s -m "Add Contributing section to Readme"```
+    ```
+     git commit -s -m "Add alerts tests"
+     ```
 
 - [Create a personal access token in Github](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) to be able to push your changes
 
 - Push your changes:  ```git push```
 
-- Send a PR to ods-ci using Github's web interface 
+- Send a PR to ods-ci using GitHub's web interface
+
+- If the PR can't be merged, rebase your branch to master:
+  ```
+  git fetch upstream
```

Comment:
would it be worth to add also the very first step `git remote add upstream https://github.com/red-hat-data-services/ods-ci` ? it is to be done only the first time

Reply:
Added!